cl/citations/description_score.py

```diff
@@ -0,0 +1,69 @@
+import re
+from math import log
+
+from cl.search.models import OpinionCluster
+
+_GERUND = r"(?:\S+ing)"
+_GERUND_THAT = rf"{_GERUND} that"
+_HOLDING = r"(?:holding|observing|deciding|ruling|recognizing)"
+_HOLDING_THAT = rf"{_HOLDING} that"
```

Comment:
I guess we should compile these regexes for a small performance boost? They'll be used millions of times.

Reply:
Same probably applies to yesterday's regexes too?

Reply:
good point, making that fix